dataland-frontend/tests/e2e/fixtures/custom_mocks/AllEuNonFinancialsDataServerResponse.ts

```diff
@@ -0,0 +1,81 @@
+import { generateCurrencyValue, generatePercentageValue } from "@e2e/fixtures/common/NumberFixtures";
+import { DEFAULT_PROBABILITY } from "@e2e/utils/FakeFixtureUtils";
+import { generateNaceCodes } from "@e2e/fixtures/common/NaceCodeFixtures";
+import { generateDataPoint, generateReferencedReports } from "@e2e/fixtures/common/DataPointFixtures";
+import { generateEuTaxonomyWithBaseFields } from "@e2e/fixtures/eutaxonomy/EuTaxonomySharedValuesFixtures";
+import { generateCurrencyCode } from "@e2e/fixtures/common/CurrencyFixtures";
+import { generateArray } from "@e2e/fixtures/FixtureUtils";
+import { range } from "@/utils/ArrayUtils";
+
+class MinimumAcceptedEuNonFinancialsGenerator extends EuNonFinancialsGenerator {
+  generateMinimumAcceptedEuTaxonomyForNonFinancialsData(): EuTaxonomyDataForNonFinancials {
+    return {
+      general: generateEuTaxonomyWithBaseFields(this.reports, this.setMissingValuesToNull, 0),
+      revenue: this.generateMinimumAcceptedDetailsPerCashFlowType(),
+      capex: this.generateMinimumAcceptedDetailsPerCashFlowType(),
+      opex: this.generateMinimumAcceptedDetailsPerCashFlowType(),
+    };
+  }
+
+  generateMinimumAcceptedDetailsPerCashFlowType(): EuTaxonomyDetailsPerCashFlowType {
+    return {
+      totalAmount: generateDataPoint(
+        this.valueOrMissing(generateCurrencyValue()),
+        this.reports,
+        this.setMissingValuesToNull,
+        generateCurrencyCode(),
+      ),
+      nonEligibleShare: this.generateFinancialShare(),
+      eligibleShare: this.generateFinancialShare(),
+      nonAlignedShare: this.generateFinancialShare(),
+      nonAlignedActivities: generateArray(() => this.generateActivity(), 1, 2),
+      alignedShare: this.generateFinancialShare(),
+      substantialContributionToClimateChangeMitigationInPercent: generatePercentageValue(),
+      substantialContributionToClimateChangeAdaptionInPercent: generatePercentageValue(),
+      substantialContributionToSustainableUseAndProtectionOfWaterAndMarineResourcesInPercent: generatePercentageValue(),
+      substantialContributionToTransitionToACircularEconomyInPercent: generatePercentageValue(),
+      substantialContributionToPollutionPreventionAndControlInPercent: generatePercentageValue(),
+      substantialContributionToProtectionAndRestorationOfBiodiversityAndEcosystemsInPercent: generatePercentageValue(),
+      alignedActivities: generateArray(() => this.generateAlignedActivity(), 1, 2),
+      enablingShareInPercent: generatePercentageValue(),
+      transitionalShareInPercent: generatePercentageValue(),
+    };
+  }
+}
+
+/**
+ * Generates a list of data and meta information with EU taxonomy for non financials data
+ * @returns a list of data and meta information with EU taxonomy for non financials data
+ */
+export function generateEuTaxonomyForNonFinancialsMocks(): DataAndMetaInformationEuTaxonomyDataForNonFinancials[] {
```

Comment:
I'd assume that this custom mock returns EUTaxoNonFinancial mocks for a specific purpose (compared to "non_custom" mocks)? What is this purpose? Can it be mentioned in the docstring?

Reply:
Changed the docstring in the index.ts file to reflect the difference of the type of mock.